frontend/web/components/base/link/Link.scss

```
@@ -0,0 +1,25 @@
+// Light mode is the same #6837fc as btn-link, so those call sites will not
+// shift when they move over. Dark deliberately differs: `.dark .btn-link` keeps
+// that same purple, this takes a lighter one for contrast on dark backgrounds,
+// so moving a link over is a visible change there.
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)